docs/Changelog.md

```diff
@@ -18939,6 +18939,49 @@ This version of the operator has been available since version 14 of the default
 <dd>Constrain seq_lens to integer tensor.</dd>
 </dl>
 
+### <a name="GridSampler-14"></a>**GridSampler-14**</a>
+
+  Given an input and a flow-field grid, computes the output using input values and pixel locations from grid.
```

Comment:
Consider if we also need to support volumetric (5-D) input support.

Reply:
There are couple of issues opened for this op. We can ask there if people need 5-D support if so we can add it now otherwise it is OK to add it later as need arises.

Reply:
According to the example of several github issues that are provided input, all the inputs are 4-D shape. I think we can support 4-D it firstly.
https://github.com/pytorch/pytorch/issues/41957
https://github.com/pytorch/pytorch/issues/27212
https://github.com/AliaksandrSiarohin/first-order-model/issues/92
https://github.com/PaddlePaddle/Paddle2ONNX/issues/273